governance/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "forta-{developerAbbreviation}-{protocolAbbreviation}-function-calls",
+  "version": "0.0.1",
+  "description": "Forta Agent template for monitoring function calls from {protocolName} contracts",
```

Comment:
Update description for governance agent template

Reply:
Done.